spec/classes/etherpad_spec.rb

```diff
@@ -58,6 +57,43 @@
     end
   end
 
+  context 'supported operating systems' do
+    on_supported_os.each do |os, facts|
+      context "on #{os}" do
+        let(:facts) do
+          facts
+        end
+
+        context 'etherpad class with plugins_list enabled' do
+          let(:params) do
+            {
+              plugins_list: {
+                'ep_ldapauth'    => true,
+                'ep_button_link' => :undef,
+                'ep_align'       => false
+              },
+              ldapauth: {
+                'url'                => 'ldap://ldap.foobar.com',
+                'accountBase'        => 'o=staff,o=foo,dc=bar,dc=com',
+                'groupAttributeIsDN' => false
+              }
+            }
+          end
+
+          it { is_expected.to contain_file('/opt/etherpad/settings.json').with_content(%r|^\s*"users": {$|) }
```

Comment:
The catalog no longer contains a `File` resource, since you're now using the `concat` module for `settings.json`.  This is why the tests are failing.

Reply:
Ok, I will try it.